docs/testing/troubleshooting-e2e.md

```diff
@@ -0,0 +1,14 @@
+# Troubleshooting E2E Tests
```

Comment:
Can you also help add the pod logs https://github.com/knative/eventing/pull/3113 and metric logs https://github.com/google/knative-gcp/pull/1069 in this troubleshooting session?

Reply:
Added pod logs. The metric logs seem to be already in the prow page IIUC.